sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -4,12 +4,14 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Changed `GalleryImageImpl` to use patch pattern when updating.
```

Comment:
```suggestion
- Changed to use PATCH for `GalleryImage` update.
```

Reply:
Do not mention Impl as this is not user will see.

Please also refresh main. CMK is merged and there is also line in `feature added`. Make sure it is still correct after refresh.